frontend/app/src/components/history/transactions/TransactionContent.vue

```diff
@@ -358,16 +361,28 @@ watch(usedAccount, async () => {
 
 const loading = isSectionLoading(Section.TX);
 const eventTaskLoading = isTaskRunning(TaskType.TX_EVENTS);
+const { isAllFinished } = toRefs(useTxQueryStatus());
 
 const { pause, resume, isActive } = useIntervalFn(() => fetch(), 10000);
 
-watch([loading, eventTaskLoading], ([sectionLoading, eventTaskLoading]) => {
-  if ((sectionLoading || eventTaskLoading) && !get(isActive)) {
-    resume();
-  } else if (!sectionLoading && !eventTaskLoading && get(isActive)) {
-    pause();
+watch(
+  [loading, eventTaskLoading, isAllFinished],
+  ([sectionLoading, eventTaskLoading]) => {
+    if (
+      (sectionLoading || eventTaskLoading || !isAllFinished) &&
```

Comment:
isAllFinished here is a `ref` so it always evaluates true right? because it is missing from 370.

Reply:
nice catch